src/Util/VerifyEmailQueryUtility.php

```diff
@@ -15,6 +15,8 @@
  * @author Jesse Rushlow <jr@rushlow.dev>
  * @author Ryan Weaver   <ryan@symfonycasts.com>
  *
+ * @deprecated since v1.17.0 - Remove in 2.0
```

Comment:
We're still ALWAYS injecting this class now, right? So maybe we need to wait to deprecate until 2.0?

Reply:
good catch